src/status_im2/contexts/shell/activity_center/view.cljs

```diff
@@ -53,26 +54,42 @@
                                            :override-theme :dark})])
                           (rf/dispatch [:hide-bottom-sheet]))}]]]))
 
+(defn empty-tab-description
+  [filter-type]
+  (case filter-type
+    types/no-type
+    (i18n/label :t/empty-notifications-all-tab)
+    types/admin
+    (i18n/label :t/empty-notifications-admin-tab)
+    types/mention
+    (i18n/label :t/empty-notifications-mentions-tab)
+    types/reply
+    (i18n/label :t/empty-notifications-replies-tab)
+    types/contact-request
+    (i18n/label :t/empty-notifications-contact-requests-tab)
+    types/contact-verification
+    (i18n/label :t/empty-notifications-identity-verification-tab)
+    types/tx
+    (i18n/label :t/empty-notifications-transactions-tab)
+    types/membership
+    (i18n/label :t/empty-notifications-membership-tab)
+    types/system
+    (i18n/label :t/empty-notifications-system-tab)
+    nil))
```

Comment:
This can be refactored into a CLJS map and we can use the `get` keyword to lookup values in it
Example
```clj
{types/no-type :t/empty-notifications-all-tab
...}
```
And we can leverage the not-found third arg of the `get `keyword
`(get filter-type empty-tab-descriptions nil)`

Reply:
Done